docs/guides/haiku_migration_guide.rst

```diff
@@ -713,4 +713,152 @@ Finally, let's quickly view how the ``RNN`` Module would be used in both Haiku a
 
 The only notable change with respect to the examples in the previous sections is that
 this time around we used ``hk.without_apply_rng`` in Haiku so we didn't have to
-pass the ``rng`` argument as ``None`` to the ``apply`` method.
\ No newline at end of file
+pass the ``rng`` argument as ``None`` to the ``apply`` method.
+
+Scan Over Layers
```

Comment:
```suggestion
Scan over layers
```

We can implement other style guide-related changes to the rest of the doc in another PR.